tools/clang/lib/SPIRV/SPIRVEmitter.h

```diff
@@ -106,6 +106,11 @@ class SPIRVEmitter : public ASTConsumer {
   SpirvEvalInfo doMemberExpr(const MemberExpr *expr);
   SpirvEvalInfo doUnaryOperator(const UnaryOperator *expr);
 
+  /// Overload with pre computed SpirvEvalInfo.
```

Comment:
Nit: would you put these two next to each other?

Reply:
This one is a `private:` one. The other one is `public:`.

Reply:
Oh sorry. yes. Never mind then. Thanks